templates/shared/forms/interactive/landscape-managed.html

```diff
@@ -89,7 +89,7 @@ <h3>Your comments</h3>
                 </div>  
   
                 <div class="pagination">
-                  <button type="submit" class="pagination__link--next p-button--positive" aria-label="Submit">Let's discuss</button>
+                  <button class="pagination__link--next p-button--positive" aria-label="Submit">Let's discuss</button>
```

Comment:
Removing the button type also removes the loading spinner animation that is supposed to prevent multiple submissions